mono/metadata/threads-types.h

```diff
@@ -55,6 +55,12 @@ typedef enum {
 
 typedef struct _MonoInternalThread MonoInternalThread;
 
+/* It's safe to access System.Threading.InternalThread from native code via a
+ * raw pointer because all instances should be pinned.  But for uniformity of
+ * icall wrapping, let's declare a MonoInternalThreadHandle anyway.
+ */
+TYPED_HANDLE_DECL (MonoInternalThread);
```

Comment:
How is this change related to perfcounter?

Reply:
Not sure, might be leftover. I'll try removing.